### DISCUSSION TITLE
ART 1.13.1

### DISCUSSION THREAD

**Discussion by beat-buesser** [Trusted-AI/adversarial-robustness-toolbox#2034] [Announcements]
This release of ART 1.13.1 provides updates to ART 1.13

# Added

[None]

# Changed

- Changed PDTP privacy metric to support two comparison: ratio (default) and new difference mode (#1984)
- Changed default parameters for `apply_fit` and `apply_predict` for the Data Augmentation defenses `CutMix*`, `CutOut*`, and `MixUp*` (#1987)

# Removed

[None]

# Fixed

- Fixed bug in `PixelThreshold` attack to support batches of a single sample (#1982)
- Fixed type error in `DPInstaHideTrainer` for `PyTorchClassifier` by casting random noise to correct type (#1987)
- Added missing classes to union types `OBJECT_DETECTOR_TYPE`, `PYTORCH_ESTIMATOR_TYPE`, and `TENSORFLOWV2_ESTIMATOR_TYPE` (#1999)
- Fixed audio perturbations going out of clip values in `insert_tone_trigger` and `insert_audio_trigger` (#2016)
- Fixed missing transfer to device in `FeatureAdversariesPyTorch` to enable running on GPUs (#2021)
- Fixed missing covnersion to float to support floor() on GPUs in `PyTorchClassifier` (#2022)
- Fixed incorrect integer return type in `check_and_transform_label_format` (#2025)

<hr /><em>This discussion was created from the release <a href='https://github.com/Trusted-AI/adversarial-robustness-toolbox/releases/tag/1.13.1'>ART 1.13.1</a>.</em>
